src/wallet.js

```diff
@@ -508,7 +508,7 @@ const wallet = {
    * @memberof Wallet
    * @inner
    */
-  loadAddressHistory(startIndex, count, connection = null, store = null) {
+  loadAddressHistory(startIndex, count, connection = null, store = null, preCalculatedAddresses = null) {
```

Comment:
We need to add this to the docstring

Reply:
Fixed on [5c74a59](https://github.com/HathorNetwork/hathor-wallet-lib/pull/354/commits/5c74a595c99e414d85ee32b924b2159c0e12f5b7)